test/e2e/packages/debian-10/n4c16/test01-systemd/code.var.sh

```diff
@@ -0,0 +1,10 @@
+# Clear cri-resmgr output from previous runs.
+vm-command "journalctl --vacuum-time=1s"
+
+# Create a pod.
+create besteffort
```

Comment:
Maybe also splitting out pulling the image to avoid `kubectl wait` timing out due to slow network giving a false failure. With something like this:
```
...
image=$(cat <path-to->besteffort.yaml | egrep '^ *image: .*$' | sed -E 's/^ *image: *([^ ]*)$/\1/g')
vm-command crictl -i unix:///var/run/cri-resmgr/cri-resmgr.sock pull $image
create besteffort
...
```
On my first try I hit exactly such a  `kubectl wait` timeout for the busybox image.

Reply:
Or to rephrase, considering the timeout snafu I hit, wouldn't it make sense to add a test API function for `pull <yaml-template-name>` which would then do that above for all images found in the template ?

```
for img in $(cat <path-to-yaml-template> | egrep '^ *image: .*$' | sed -E 's/^ *image: *([^ ]*)$/\1/g'); do
    echo "Pre-fetching image $img for <path-to-yaml-template>..." 
    vm-command crictl -i unix:///var/run/cri-resmgr/cri-resmgr.sock pull $img
done
```

Reply:
We're using busybox image in all tests, so I'd be tempted to add

```
crictl -i unix:///var/run/cri-resmgr/cri-resmgr.sock pull $image
```

as part of the test setup in infra. What do you think?

Reply:
I think that should be good enough for now. We can make it more flexible later if/when ever needed.

Reply:
Ended up embedding this into `create()` function. It pulls all (not-pulled-earlier) images it finds in instantiated templates before `kubectl create`.